x-pack/plugin/enrich/src/main/java/org/elasticsearch/xpack/enrich/action/TransportDeleteEnrichPolicyAction.java

```diff
@@ -39,7 +39,9 @@ protected String executor() {
 
     @Override
     protected AcknowledgedResponse newResponse() {
-        throw new UnsupportedOperationException("usage of Streamable is to be replaced by Writeable");
+        return new AcknowledgedResponse();
```

Comment:
I'm working on a PR that changes AcknowledgedResponse and subclasses from streamable to writable. It is large change, but it is mostly mechanical.

Reply:
I've opened this PR #43414